server/routerlicious/packages/routerlicious/config/telegraf/telegraf.conf

```diff
@@ -89,8 +89,8 @@
   ## Write timeout (for the InfluxDB client), formatted as a string.
   ## If not provided, will default to 5s. 0s means no timeout (not recommended).
   timeout = "5s"
-  # username = "telegraf"
-  # password = "metricsmetricsmetricsmetrics"
+  # username = "PLACEHOLDER"
```

Comment:
Can we add a comment in this file that mentions the special functionality for the `PLACEHOLDER` keyword with regards to the cred suppression task